tests/foreman/cli/test_contentview.py

```diff
@@ -1231,6 +1233,57 @@ def test_positive_add_rh_repo_by_id_and_create_filter(self):
             'name': 'walgrind',
         })
 
+    @run_in_one_thread
+    @tier3
+    @upgrade
+    def test_positive_add_module_stream_filter_rule(self):
+        """Associate module stream content to a content view and create filter rule
+
+        :id: 8186a4b2-1c11-11ea-99cf-d46d6dd3b5b2
+
+        :setup: Sync custom yum repo content
+
+        :steps: Assure filter(s) applied to associated content
+
+        :expectedresults: Filter rule should get applied to Filter
+
+        :CaseImportance: Low
+
+        :CaseLevel: Integration
+
+        """
+        filter_name = gen_string('alpha')
+        repo_name = gen_string('alpha')
+        org = entities.Organization().create()
```

Comment:
`setUpClass` is creating same :
https://github.com/SatelliteQE/robottelo/blob/6969daac216f48582eaf4c172fa528e8dc7a45f6/tests/foreman/cli/test_contentview.py#L171